dis_snek/models/snek/wait.py

```diff
@@ -17,8 +17,8 @@ def __call__(self, *args, **kwargs) -> bool:
         if self.checks:
             try:
                 check_result = self.checks(*args, **kwargs)
-            except Exception:
-                return True
+            except Exception as exc:
+                raise exc
```

Comment:
Just remove the whole `try`-statement at this point.

Reply:
lol, okay